packages/query/src/graphql/plugins/GetMetadataPlugin.ts

```diff
@@ -47,20 +60,13 @@ async function fetchFromApi(): Promise<void> {
   }
 }
 
-async function fetchFromTable(context: any, schemaName: string): Promise<Metadata> {
-  const metadata: Metadata = {
-    lastProcessedHeight: 0,
-    lastProcessedTimestamp: 0,
-    targetHeight: 0,
-    chain: '',
-    specName: '',
-    genesisHash: '',
-    indexerHealthy: false,
-    indexerNodeVersion: '',
-    queryNodeVersion: '',
-  };
+async function fetchFromTable(pgClient: any, schemaName: string): Promise<Metadata> {
+  const metadata = {} as Metadata;
+
+  const keys = Object.keys(METADATA_TYPES);
+  const formattedKeys = `'${keys.join("','")}'`;
 
-  const {rows} = await context.pgClient.query(`select key, value from ${schemaName}._metadata`);
+  const {rows} = await pgClient.query(`select key, value from ${schemaName}._metadata WHERE key IN (${formattedKeys})`);
```

Comment:
use Parameterized query https://node-postgres.com/api/client

Reply:
managed to get it to work with `WHERE key = ANY`